Languages/french.lang

```diff
@@ -183,7 +183,7 @@ msgid "Are you sure you want to lock USB Loader GX?"
 msgstr "Voulez-vous vraiment verrouiller USB loader GX ?"
 
 msgid "Are you sure you want to remount SD?"
-msgstr "Voulez-vous recharger la carte SD ?"
+msgstr "Voulez-vous vraiment recharger la carte SD ?"
```

Comment:
I would say «remonter» – in unix/linux circles, we use _monter_ for _mount_ (filesystems)